.github/workflows/daemon-swift.yml

```
@@ -0,0 +1,64 @@
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: jdx/mise-action@v2
+
+      - name: Restore SPM cache
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)